### DISCUSSION TITLE
JK BMS Multiple Battery RS485

### DISCUSSION THREAD

**Discussion by RichGorrod** [Louisvdw/dbus-serialbattery#1119] [Q&A]
Hello,
I am would like to connect 4 batteries with JK invertor BMS to a Cebo GX via RS485. - I can connect 1 no problem and working well. Is there anything I need to do to to connect all 4. Ideally linking them together and addressing them differently. 1,2,3,4.

If I use Can bus for the master and RS485 linking between batteries works fine but I cant see the individual cell voltage in Node red.

Any help would be great

Many thanks

Rich

**Comment by cmock** [comment]
I have two JKBMS on an RS485 connection:

- connected Cerbo to first BMS' RS485-2, daisychained the second
- you cannot set any BMS to address 0 (DIP switches), I set them to 1 and 2
- config.ini:
```
MODBUS_ADDRESSES = 0x01, 0x02
BMS_TYPE = Jkbms_pb
```

works fine, they show up in the remote console as 2 batteries, which are then combined via dbus-aggregate-batteries into the one virtual battery the system uses.

Edit: I'm using the new "inverter BMSes", PB2A16S20P.
